lib/search/searchSnippet.ts

```
@@ -0,0 +1,66 @@
+/**
+ * Creates a smart text snippet that shows context around matched search terms
+ * Returns the original text if the match is early enough to be shown with normal truncation
+ */
+export function createSearchSnippet(
+  text: string,
+  searchTerms: string[],
+  maxLength: number = 150
+): string {
```

ESLint

[error] Replace `⏎··text:·string,⏎··searchTerms:·string[],⏎··maxLength:·number·=·150⏎` with `text:·string,·searchTerms:·string[],·maxLength:·number·=·150`

(prettier/prettier)